README.md

```diff
@@ -441,7 +441,7 @@ submissions from the system:
 
 We recommend `USESAFEDOCKER`, as that is what we test in practice.
 
-When using `safe-docker` while Praktomat itself is already running inside a Docker container, you need to have a BusyBox image available. It is required for some temporary containers.
+If you are using Docker to run external commands and Praktomat itself is already running inside a Docker container, you need to have a BusyBox image available. It is required for some temporary containers.
```

Comment:
It would be better to state what you need from BusyBox and then state that BusyBox provides all requirments.

Reply:
The reason for this is that the `busybox` image is hard-coded in `safeexec.py`. Other images could be used in theory, but then the code would need to be adjusted as well.

Reply:
Ok!